main.py

```diff
@@ -1,25 +1,69 @@
 import os
 import json
 
+import argparse
 from app.modbus_client import ModbusClient
 from app.mqtt_client import MqttClient
-from app.configuration import Configuration
+from app.configuration import Configuration, ModbusSettings, MqttSettings 
 
 SLAVE=0x01
 
-BROKER_HOST = os.getenv('BROKER_HOST')
-BROKER_PORT = int(os.getenv('BROKER_PORT'))
-MODBUS_HOST = os.getenv('MODBUS_HOST')
-MODBUS_PORT = int(os.getenv('MODBUS_PORT'))
-COMMANDS_TOPIC = os.getenv('COMMANDS_TOPIC')
-CONFIGURATION_PATH = os.getenv('CONFIGURATION_PATH', "config/example_configuration.yaml")
+def handle_args():
+    # Create the argument parser
+    parser = argparse.ArgumentParser(description='remote commands handler')
+
+    # Add the optional arguments
+    parser.add_argument('--configuration_path', required=True, help='Configuration path')
```

Comment:
This one should have a default value pointing to the `config/configuration.yaml` file.

Reply:
```suggestion
    parser.add_argument('--configuration_path', required=True, help='Configuration path', default='config/configuration.yaml')
```

I guess